packages/builders/src/node/build/webpack/config.ts

```diff
@@ -46,7 +46,7 @@ export function getWebpackConfig(
       ]
     },
     resolve: {
-      extensions: ['.ts', '.js'],
+      extensions: ['.ts', '.js', '.mjs'],
```

Comment:
So I tested it and it works!

However, should `.mjs` precede `.js`?

This would make it so that in cases where both exist
```
index.js
index.mjs
```
`.mjs` should be the one which is used?

What do you think?

Reply:
Some libraries are packed that way, like `graphql`. Webpack tries to use them, as it is easier and more optimal to bundle them, than let's say `umd`. I think webpack optimizes the size of output as well.
I don't think it is matter of precedence. Webpack in anyway tries to load `mjs` if it is there, and when it is not configured, build get screwed.
Also I don't think your comments are relevant for this case. Yes, it is not about project `mjs`, it is about being able to load external dependencies.
In my case, I bundle my backends in single bundle, so that I can dockerize them easily without need to bring whole `node_modules` inside container, which is very painful. I'm also not using `externalDependencies` unless necessary (only if there is binary dependency, it is going to `externalDependencies`.
Without `mjs` extension, I had to put such externals to `externalDependencies`, in order to avoid them from bundling. When you add `mjs` to webpack config, webpack happily bundles them.

Reply:
Okay thanks, we are on the same page 👍.

I think the order does matter in terms of which extension gets precedence for bundling. Can we change the order so it matches the default order more closely please?

https://webpack.js.org/configuration/resolve/

```suggestion
      extensions: ['.ts', '.mjs', '.js'],
```